akka-actor/src/main/resources/reference.conf

```diff
@@ -533,6 +534,25 @@ akka {
       mailbox-requirement = ""
     }
 
+    # The full config path of a dispatcher to use for all internal actors to
+    # avoid starvation in the default dispatcher affecting Akka internals
+    internal-dispatcher = "akka.actor.default-internal-dispatcher"
+
+    # Default separate internal dispatcher, if 'akka.actor.internal-dispatcher' is changed this
+    # dispatcher is not used
+    default-internal-dispatcher {
+      type = "Dispatcher"
+      executor = "fork-join-executor"
+      throughput = 5
+      fork-join-executor {
+        parallelism-min = 4
+        parallelism-factor = 1.0
+        # FIXME no use to have a high number here, but what is reasonable?
+        parallelism-max = 8
```

Comment:
shouldn't matter much but might be surprising that a large box is limited to 8? maybe 64, and actually increase it for default-dispatcher to 256?

Reply:
Alright, I only have vague gut feeling saying there won't be very many internal actors but I guess if we run sharding for example, and a few shard types and a bunch of shard actors which will run on internal...

Reply:
I also have some concerns about these numbers but maybe let's keep these out of this PR and add another issue to tweak default settings? It will need a few considerations and benchmarks to arrive at something reasonable. Also we should probably document a few scenarios that we can think of and how to configure for them (performance -> only one dispatcher, etc).